packages/styles/scss/components/treeview/_treeview.scss

```
@@ -135,15 +135,15 @@
       min-width: 1rem;
       min-height: 1rem;
       margin-right: $spacing-03;
-      fill: $icon-02;
+      fill: $icon-secondary;
     }
 
     .#{$prefix}--tree-node--selected > .#{$prefix}--tree-node__label {
-      background-color: $selected-ui;
-      color: $text-01;
+      background-color: $layer-selected-01;
+      color: $text-primary;
 
       &:hover {
-        background-color: $hover-selected-ui;
+        background-color: $background-selected-hover;
```

It should continue to use the `layer` tokens instead of the `background` ones.
```suggestion
        background-color: $layer-selected-hover-01;
```